nautobot/core/views/mixins.py

```diff
@@ -995,7 +995,7 @@ def send_bulk_delete_objects_to_job(self, request, pk_list, model, delete_all):
     def send_bulk_edit_objects_to_job(self, request, form, model):
         """Prepare and enqueue a bulk edit job."""
         job_model = Job.objects.get_for_class_path(BulkEditObjects.class_path)
-        form_data = normalize_querydict(request.POST, form)
+        form_data = normalize_querydict(request.POST, form_class=form)
```

Comment:
Looking at this again, I'm wondering if this call is even needed - normally the whole point of `normalize_querydict` is for use with `request.GET` since there's no easy way to distinguish between single- and multi-value query params. For a form-encoded POST request, the form itself should already have the correct single- versus multi-value representation, shouldn't it?

Reply:
It seems like passing `request.POST` directly to the job does not work, it still makes the job fail with the same error. we still need the `normalize_query_dict` to convert the Querydict object to a dict. I am starting to feel like maybe the problem lies within `TagsBulkEditFormMixin`

Reply:
Okay, I think the right fix is to instantiate the `form_class` when it is not null, calling the `__init__` method and checking the `fields` attribute. https://github.com/nautobot/nautobot/pull/6824/commits/b286940e228a1f670393fc274874454cf482d0c7

Reply:
Latest CI failure suggests that `form` is already an instance rather than a class?

Reply:
https://github.com/nautobot/nautobot/pull/6824/commits/ff4205507992b0a81d58c23d7aad8fc65f8e50c2 Yeah added some parameter type enforcement for `normalize_querydict` method. Hopefully it is not a breaking change for apps.